lib/ui/painting/image_decoding.cc

```diff
@@ -34,24 +34,13 @@ sk_sp<SkImage> DecodeImage(sk_sp<SkData> buffer) {
     return nullptr;
   }
 
-  auto raster_image = SkImage::MakeFromEncoded(std::move(buffer));
-
-  if (raster_image == nullptr) {
-    return nullptr;
-  }
-
-  if (auto context = ResourceContext::Get()) {
-    // TODO: Supply actual destination color space once available
```

Comment:
Can be keep this comment please? Adding it next to the last argument of `SkImage::MakeCrossContextFromEncoded`.